### PR TITLE
fix link in comment

### PR DESCRIPTION
**Pull request by GKuldeepak-Knoldus**
Fixed External Links

References #30697

### CODE REVIEW

**Review by ennru — APPROVED** [review]
LGTM.

File: akka-protobuf/src/main/java/akka/protobuf/RpcChannel.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 // Protocol Buffers - Google's data interchange format
 // Copyright 2008 Google Inc.  All rights reserved.
-// http://code.google.com/p/protobuf/
+// https://github.com/protocolbuffers/protobuf
```
Comment (raboof) [inline akka-protobuf/src/main/java/akka/protobuf/RpcChannel.java:7]: I'm not sure we should change URL's in copyright headers... if we do update it, I think we should follow upstream (https://github.com/protocolbuffers/protobuf/blob/master/java/core/src/main/java/com/google/protobuf/RpcChannel.java) and change it to https://developers.google.com/protocol-buffers/

Comment (ennru) [reply akka-protobuf/src/main/java/akka/protobuf/RpcChannel.java:7]: Yeah, you're right. Let's stick to the sourced files.

Comment (patriknw) [reply akka-protobuf/src/main/java/akka/protobuf/RpcChannel.java:7]: @GKuldeepak-Knoldus could you revert the changes to the copyright headers, pls

Comment (GKuldeepak-Knoldus) [reply akka-protobuf/src/main/java/akka/protobuf/RpcChannel.java]: @ennru , I have reverted the changes as suggested by you. Please have a look.

File: akka-protobuf/src/main/java/akka/protobuf/ServiceException.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 // Protocol Buffers - Google's data interchange format
 // Copyright 2008 Google Inc.  All rights reserved.
-// http://code.google.com/p/protobuf/
+// https://github.com/protocolbuffers/protobuf
```
Comment (raboof) [inline akka-protobuf/src/main/java/akka/protobuf/ServiceException.java:7]:
```suggestion
// http://code.google.com/p/protobuf/
```

File: akka-protobuf/src/main/java/akka/protobuf/UnknownFieldSet.java [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 // Protocol Buffers - Google's data interchange format
 // Copyright 2008 Google Inc.  All rights reserved.
-// http://code.google.com/p/protobuf/
+// https://github.com/protocolbuffers/protobuf
```
Comment (raboof) [inline akka-protobuf/src/main/java/akka/protobuf/UnknownFieldSet.java:7]:
```suggestion
// http://code.google.com/p/protobuf/
```

**Review by raboof — COMMENTED** [review]
I think these two are probably considered part of Google's copyright header as well...

**Review by jrudolph — APPROVED** [review]
LGTM

**Pull request merged**